index.html

```diff
@@ -2120,588 +2138,378 @@ <h3>Thing Root Object</h3>
+    instance of <code>SecurityScheme</code>, MUST be serialized as a JSON object.
   </span></p>
 
-  <p><span class="rfc2119-assertion" id="td-uriVariables-dataschema">
-  The serialization of each value in the map assigned to <code>uriVariables</code>
-  in an instance of <code>Form</code> MUST
-  rely on the <a>Class</a> <a href="#dataschema" class="sec-ref"><code>DataSchema</code></a>,
-  whose serialization is defined in <a href="#data-schema-serialization-json"></a>.
+  <p><span class="rfc2119-assertion" id="td-security-schemes">
+    All name-value pairs of an instance of one of the <a>Subclasses</a> of
+    <code>SecurityScheme</code>,
+    where the name is a <a>Vocabulary Term</a> included in the
+    <a>Signature</a> of that <a>Subclass</a> or in the <a>Signature</a> of
+    <code>SecurityScheme</code>, MUST be serialized as members of the JSON object
+    that results from serializing the <code>SecurityScheme</code> <a>Subclass</a>'s
+    instance, with the <a>Vocabulary Term</a> as name.
   </span></p>
 
-  <p>A TD snippet using a URI Template and <code>uriVariables</code> is given below:</p>
+  <p>
+  The following TD snippet shows a simple security configuration specifying
+  basic username/password authentication in the header.
+  The value given for <code>in</code> is actually the <a>Default Value</a> (<code>header</code>)
+  and could be omitted.
+  First, a named security configuration must be given
+  in the <code>securityDefinitions</code> map.
+  Second, that definition must be activated by inclduing its JSON name in the
+  <code>security</code> array.
+  </p>
 
-<pre class="example">
-{
-    "@context": [
-        "https://www.w3.org/2019/td/v1",
-        { "eg": "http://www.example.org/iot" }
-    ],
+<pre class="example" id="security-basic-example">
     ...
-    "actions": {
-        "LeftDown": {
-            ...
-            "uriVariables": {
-                "p" : { "type": "integer", "minimum": 0, "maximum": 16, "@type": "eg:SomeKindOfAngle" },
-                "d" : { "type": "integer", "minimum": 0, "maximum": 1, "@type": "eg:Direction" }
-            },
-            "forms": [{
-              "href" : "http://192.168.1.25/left{?p,d}",
-              "htv:methodName": "GET"
-            }]
-        },
-        ...
+    "securityDefinitions": {
+        "basic_sc": {
+            "scheme": "basic",
+            "in": "header"
+        }
     },
+    "security": ["basic_sc"],
     ...
-}
 </pre>
 
   <p>
-  The <code>contentType</code> member is used to assign a media types [[!IANA-MEDIA-TYPES]]
-  including media type parameters as attribute-value pairs separated by a <code>;</code> character.
-  Example:
+  Here is a more complex example: a TD snippet showing digest authentication
+  on a proxy combined with bearer token authentication on the <a>Thing</a>.
+  Here the <a>Default Value</a> of <code>in</code> in the <code>digest</code> scheme,
```

Comment:
This sentence is not very clear, please reword.
What you want to say, is that it is not required to explicitly include default values.

Reply:
Same text as ever since, only marked because moved.

Reply:
Maybe say "given in" to avoid having two "in"s in a row...

Reply:
In the <code>digest</code> scheme, the <a>Default Value</a> of <code>in</code>
  (i.e., <code>header</code>) is omitted, but still applies.